compute/instances/custom-machine-type/utils.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+func containsString(s []string, str string) bool {
```

Comment:
Can this be moved to where it's used?

Reply:
I am going to use the function in other samples in the same folder. What's the best way to do the same logic in different files in one package (I guess to rename the function in every file is super wired weird)

Reply:
Since it's such a simple function, I think duplicating it makes sense, probably by declaring it inline where necessary. That way, sample reads see the entire code in a single region tag.

For example:

``` golang
func (t customMachineType) String() string {
        containsString := func(s []string, str string) bool {
            // ...
        }
	if containsString([]string{e2Small, e2Micro, e2Medium}, t.cpuSeries) {
		return fmt.Sprintf("zones/%v/machineTypes/%v-%v", t.zone, t.cpuSeries, t.memoryMb)
	}
        // ...
}
```